dashboard/app/models/teacher_feedback.rb

```diff
@@ -90,6 +91,38 @@ def self.latest
     find_by(id: maximum(:id))
   end
 
+  def user_level
+    @user_level ||= user_levels.where(level_id: level_id, script_id: script_id)&.first
+  end
+
+  def student_seen_feedback
+    return seen_on_feedback_page_at if seen_on_feedback_page_at && seen_on_feedback_page_at > created_at
+    return student_last_visited_at if student_last_visited_at && student_last_visited_at > created_at
+  end
+
+  def student_updated_since_feedback?
+    user_level.present? && user_level.updated_at > created_at
+  end
+
+  # TODO: update to use camelcase
```

Comment:
This PR is already huge, so I didn't want to do the conversion to camelCase here

Reply:
we should discuss what we want our conventions to be around this at some point. with few exceptions, our convention seems to be that we always return data from the server in snake_case and convert to camelCase on the frontend when (and if) we parse it. but i'm not convinced that's the best convention to stick to.

i would say though that since our current convention is to _not_ use camelCase on the backend, this is one of those TODO's that could end up sticking around for a long time, so might be better to track in a jira ticket rather than in an inline comment like this

Reply:
If our current convention is to send snake_case from the back-end, I say we stick with that for now.  (Apologies if it was my comment that led to this, I didn't realize that was our current convention.)

Reply:
Right now this is the method that we're using to format data and pass it to the frontend which is why I want to convert it to camelCase. But I agree with you, we need a pattern for how we go about serializing data for our frontend and for our casing in general. Other summarize methods that we have don't have a clear pattern, some are snake and some are camel case and some are both (https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/script.rb#L1413) 😬

Reply:
I'll think about what the right move is here, maybe I'll remove the TODO and just leave snake case until we come up with a pattern.